sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -487,6 +487,32 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
         """.stripMargin)
       .load(testFile("test-data/postgresql/tenk.data"))
       .createOrReplaceTempView("tenk1")
+
+    session
+      .read
+      .format("csv")
+      .options(Map("delimiter" -> "\t", "header" -> "false"))
+      .schema(
+        """
+          |unique1 int,
+          |unique2 int,
+          |two int,
+          |four int,
+          |ten int,
+          |twenty int,
+          |hundred int,
+          |thousand int,
+          |twothousand int,
+          |fivethous int,
+          |tenthous int,
+          |odd int,
+          |even int,
+          |stringu1 string,
+          |stringu2 string,
+          |string4 string
+        """.stripMargin)
+      .load(testFile("test-data/postgresql/onek.data"))
+      .createOrReplaceTempView("onek2")
```

Comment:
If this is for only `select.sql`, can we simply like the following? Maybe, we can have the following line at line 464.
```scala
session.table("onek").createOrReplaceTempView("onek2")
```

Reply:
`onek2` used in 5 files, but I do not have plan to port other files.
![image](https://user-images.githubusercontent.com/5399861/61010852-899c4b00-a3aa-11e9-8dae-5dbfd4df7818.png)

Reply:
I add `create or replace temporary view onek2 as select * from onek;` to select.sql.

Reply:
Thanks.